xbmc/video/PlayerController.cpp

```diff
@@ -74,24 +74,23 @@ bool CPlayerController::OnAction(const CAction &action)
 
         bool subsOn = !appPlayer->GetSubtitleVisible();
         appPlayer->SetSubtitleVisible(subsOn);
-        std::string sub;
-        if (subsOn)
-        {
-          std::string lang;
-          SubtitleStreamInfo info;
-          appPlayer->GetSubtitleStreamInfo(CURRENT_STREAM, info);
-          if (!g_LangCodeExpander.Lookup(info.language, lang))
-            lang = g_localizeStrings.Get(13205); // Unknown
 
-          if (info.name.length() == 0)
-            sub = lang;
-          else
-            sub = StringUtils::Format("{} - {}", lang, info.name);
-        }
+        std::string sub, lang, caption = g_localizeStrings.Get(287); /*Subtitles*/
+        SubtitleStreamInfo info;
+        appPlayer->GetSubtitleStreamInfo(appPlayer->GetSubtitle(), info);
+        if (!g_LangCodeExpander.Lookup(info.language, lang))
+          lang = g_localizeStrings.Get(13205); /*Unknown*/
+
+        if (info.name.length() == 0)
+          sub = lang;
         else
-          sub = g_localizeStrings.Get(1223);
-        CGUIDialogKaiToast::QueueNotification(CGUIDialogKaiToast::Info,
-                                              g_localizeStrings.Get(287), sub, DisplTime, false, MsgTime);
+          sub = StringUtils::Format("{} - {}", lang, info.name);
+
+        if (!subsOn)
+          caption =
+              StringUtils::Format("{}({})", caption, g_localizeStrings.Get(1223)); /*Disabled*/
```

Comment:
I think there's a space missing:
```suggestion
              StringUtils::Format("{} ({})", caption, g_localizeStrings.Get(1223)); /*Disabled*/
```